CONTRIBUTORS.md

```diff
@@ -35,7 +35,7 @@
  - [geilername](https://github.com/geilername)
  - [pR0Ps](https://github.com/pR0Ps)
  - [artiume](https://github.com/Artiume)
-
+ - [SegiH](https://github.com/SegiH)
```

Comment:
Added @SegiH as a contributor here, as the author of the original PR this one is based on: https://github.com/jellyfin/jellyfin/pull/1916